src/raw/cstr.rs

```diff
@@ -2,7 +2,30 @@ use core::str;
 
 use crate::error::{Error, Result};
 
-// A BSON-spec cstring: Zero or more UTF-8 encoded characters, excluding the null byte.
+#[allow(rustdoc::invalid_rust_codeblocks)]
```

Comment:
This was needed because otherwise rustdoc would _also_ error out when parsing the invalid string literal in the example.